src/components/Card/index.css

```diff
@@ -0,0 +1,44 @@
+.card-course {
+  padding-bottom: 10px;
+  max-width: 380px;
+}
+.card-course:before,
+.card-course:after {
+  content: '';
+  position: absolute;
+  display: block;
+  height: 10px;
+  border-radius: 5px;
+}
+.card-course:before {
+  bottom: 5px;
+  left: 9px;
```

Comment:
it's fine to deviate from the design where it makes sense for implementation. Just want to make sure that is common knowledge. Design is a guideline, we don't care about pixel perfect match.